modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,105 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, auto-instrumentation configures only the OTLP Exporter. So you must configure the endpoint to point to the OTLP Receiver on the Collector.
+
+.Sample exporter TLS CA configuration using a config map
+[source,yaml]
+----
+# ...
+spec
+  exporter:
+    endpoint: https://production-collector.observability.svc.cluster.local:4317  # <1>
+    tls:
+      configMapName: ca-bundle  # <2>
+      ca_file: service-ca.crt # <3>
+# ...
+----
+<1> Specifies the OpenTelemetry Protocol (OTLP) endpoint using the HTTPS scheme and TLS.
+<2> Specifies the name of the config map. The config map must already exist in the namespace of the pod injecting the auto-instrumentation.
```

Comment:
config map or `config map`? etc as needed

Reply:
Isn't 
config map = the `ConfigMap` object
?

Reply:
The callout appears to be clear enough:

> The config map must already exist in the namespace of the pod injecting the auto-instrumentation.